examples/classifier_example.py

```diff
@@ -13,3 +16,16 @@ def main():
 
 if __name__ == "__main__":
     main()
+
+""" Decision tree classifier
+"""
+
+def main():
```

Comment:
The function itself looks great but you can't also call it main() or else Python won't know which function to call. I would probably just move these lines into the other main() function and leave them commented out.